### PR TITLE
[Genomic Browser] add the minor_allele column after reference base

### PR DESCRIPTION
**Pull request by xlecours**
Also allow to insert allele_a and Allele_b instead of a single ObservedBase

### CODE REVIEW

File: SQL/0000-00-00-schema.sql [hunk 1]
```diff
@@ -1899,16 +1900,20 @@ CREATE TABLE `SNP` (
 --
 DROP TABLE IF EXISTS `SNP_candidate_rel`;
 CREATE TABLE `SNP_candidate_rel` (
+  `RelationID` bigint(20) NOT NULL AUTO_INCREMENT,
```
Comment (christinerogers) [inline SQL/0000-00-00-schema.sql]: ID or SNP_candidate_rel_ID would be a better name instead of "RelationID"

Comment (xlecours) [inline SQL/0000-00-00-schema.sql]: done in https://github.com/aces/Loris/pull/2183

File: SQL/Archive/17.0/2016-09-13-Add_SNP_MinorAllele.sql [hunk 2]
```diff
@@ -0,0 +1,16 @@
+ALTER TABLE SNP 
+ADD COLUMN `MinorAllele` enum('A','C','T','G') DEFAULT NULL AFTER `ReferenceBase`;
+
+ALTER TABLE SNP_candidate_rel
+CHANGE COLUMN `ObservedBase` `AlleleA` ENUM('A','C','T','G') NULL DEFAULT NULL ,
+ADD COLUMN `RelationID` BIGINT NOT NULL AUTO_INCREMENT FIRST,
```
Comment (xlecours) [inline SQL/Archive/17.0/2016-09-13-Add_SNP_MinorAllele.sql]: done in https://github.com/aces/Loris/pull/2183

**Comment by xlecours** [comment]
done in https://github.com/aces/Loris/pull/2183

**Pull request closed without merging**